tests/Sabre/Xml/ServiceTest.php

```diff
@@ -182,9 +182,6 @@ public function testEmptyPropfind(string $xml): void
         $util->namespaceMap = [
             'http://sabre.io/ns' => 'stdClass',
         ];
-        /**
-         * @var PropFindTestAsset
-         */
```

Comment:
It's better not to tell the test code what `$result` should be.
The test code does asserts itself to check that.